src/widget/wsearchrelatedtracksmenu.cpp

```diff
@@ -78,6 +78,7 @@ void WSearchRelatedTracksMenu::addTriggerSearchAction(
                     elidableTextSuffix);
     addAction(
             mixxx::escapeTextPropertyWithoutShortcuts(elidedActionText),
+            this,
```

Comment:
Good catch :+1: This is easy to get wrong. Though here it is not essential, because everything happens in the UI thread.